ckanext/nextgeoss/templates/package/base.html

```diff
@@ -29,7 +29,9 @@
       {% set thumbnail = h.ng_get_dataset_thumbnail_path(pkg) %}
       {% set col_title = pkg.title %}
       {% set col_name = h.get_extras_value(pkg.extras, 'collection_name') %}
-      {% set col_link = h.get_collection_url(col_name) %}
+      {% if col_name | length > 0 %}
```

Comment:
If `get_extras_value` would return `None` when there's no `collection_name` using 
`if col_name is not None` here would have the same effect without the drawback noted above.